### DISCUSSION TITLE
Pin mapping esp32 s3 w5500

### DISCUSSION THREAD

**Discussion by firehawk1979** [hoylabs/OpenDTU-OnBattery#1384] [Q&A]
Hallo vielleicht kann mir jemand bitte helfen, ich bekomme das mapping nicht hin . Es sollen verwendet werden esp32s3 w5500 mit cmt2300 und 2.42 Zoll Oled Display, evtl. zusätzlich ein nrf24 ( dieser muss aber nicht unbedingt sein wenn es zu kompliziert wird für mich.  Bin Anfänger und lerne ständig dazu, ich bedanke mich schonmal im voraus. 

Mit freundlichen Grüßen 

Marcel

**Comment by stefan123t** [comment]
@firehawk1979 schau Dir doch mal die Belegung / das pin_mapping.json für das OpenDTU Fusion Board an, das hat mW alle von Die gewünschten Komponenten verbaut.
Warum besorgst Du Dir nicht ein fertiges Board zB das OpenDTU-OnBattery Pro PCB oder das og Fusion Board ?

**Reply by firehawk1979** [reply]
Danke dir für die Antwort, aber es geht mir darum das ich Mithilfe des Forums lernen möchte wie man es selbst machen kann.Ansonsten hätte ich auch ein fertiges Board gekauft.

Mit freundlichen Grüßen 

Marcel

**Reply by stefan123t** [reply]
Dann ist Deine Frage evtl. besser im upstream Projekt OpenDTU bzw. im Discord aufgehoben, wenn Du eine konkrete Frage hast.

Das `pin_mapping.json` mit den GPIO Belegungen findest Du hier: https://github.com/tbnobody/OpenDTU/tree/master/docs/DeviceProfiles und die ausgezeichnete Anleitung hier: https://www.opendtu.solar/firmware/device_profiles/

**Reply by firehawk1979** [reply]
Vielen Dank genau das habe ich gesucht, werde mich morgen damit befassen. Dankeschön nochmals .😊😊😊😊

Mit freundlichen Grüßen 

Marcel

**Reply by firehawk1979** [reply]
Nochmals vielen Dank, habe es hinbekommen. ![Screenshot_20241114_002213_Samsung Internet.jpg](https://github.com/user-attachments/assets/be9b0f8f-56bc-4cfd-bda3-bb63d32bf898)

**Reply by Nessi01** [reply]
Hallo ich habe ein Esp32 -S3-Wroom 1.

Mit 44 Pin, welches pin mapping benötige ich dafür
